app/controllers/v1/actions_controller.rb

```diff
@@ -16,8 +16,11 @@ def index
   end
 
   def create
+    # need to handle invalid person
     activity = Activity.find_or_create_by(template_id: activity_param)
-    @person = person_from_params
+    @person = if person_params.any? # temporary fix
```

Comment:
good point. I forgot that instance variables will return nil if not previously defined.